local-cli/runAndroid/runAndroid.js

```diff
@@ -27,7 +27,7 @@ function checkAndroid(root) {
  */
 function runAndroid(argv, config, args) {
   if (!checkAndroid(args.root)) {
-    console.log(chalk.red('Android project not found. Maybe run react-native android first?'));
+    console.log(chalk.red('Android project not found. Maybe run react-native eject first?'));
```

Comment:
react-native eject only applies when a project has been created using create-react-native-app. We still support using `react-native init` to create a project.